.github/workflows/static-test.yml

```diff
@@ -0,0 +1,21 @@
+name: static-tests
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
```

Comment:
Can this be renamed no not be `build` somehow? It is not a build after all, but a bunch of static tests.